apps/webapp/src/components/send/send-form.tsx

```diff
@@ -1,16 +1,22 @@
 import { Button } from '@penumbra-zone/ui';
 import { useStore } from '../../state';
-import { sendSelector, sendValidationErrors } from '../../state/send';
+import { totalGasPriceSelector, sendSelector, sendValidationErrors } from '../../state/send';
 import { useToast } from '@penumbra-zone/ui/components/ui/use-toast';
 import { InputBlock } from '../shared/input-block.tsx';
 import InputToken from '../shared/input-token.tsx';
 import { LoaderFunction, useLoaderData } from 'react-router-dom';
 import { AccountBalance, getBalancesByAccount } from '../../fetchers/balances.ts';
 import { useMemo } from 'react';
 import { penumbraAddrValidation } from './helpers.ts';
+import { getGasPrices } from '../../fetchers/gas-prices.ts';
 
 export const SendAssetBalanceLoader: LoaderFunction = async (): Promise<AccountBalance[]> => {
-  const balancesByAccount = await getBalancesByAccount();
+  const [balancesByAccount, gasPrices] = await Promise.all([
+    getBalancesByAccount(),
+    getGasPrices(),
+  ]);
+
+  useStore.getState().send.setGasPrices(gasPrices);
```

Comment:
I'm using the loader to load gas prices and set them in the state. 

Per [the docs](https://reactrouter.com/en/main/route/loader), loaders should make a request, then _return_ the result, thereby making that result available to the component via `useLoaderData()`. However, the existing code was using this loader to fetch data, potentially assign it to Zustand state, and return the data.

That seems perfectly acceptable to me, even if that wasn't `react-router-dom`'s original intent with loaders, so I'm doing the same thing as well. (An alternative would be to wrap this loading in a `useEffect`, but that would prevent SSR from making these calls before the component has rendered.) But please let me know if y'all think this is an OK pattern.